diff -urN vfu-1.51.orig/ftparc/makefile vfu-1.51/ftparc/makefile
--- vfu-1.51.orig/ftparc/makefile	Sat Aug 14 17:03:15 1999
+++ vfu-1.51/ftparc/makefile	Sat Aug 14 17:05:43 1999
@@ -3,8 +3,8 @@
 CC      = g++
 LD      = g++
 AR      = ar rvs
-CFLAGS  =  -I../vslib -O2     $(CCDEF)
-CCFLAGS =  -I../vslib -O2     $(CCDEF)
+CFLAGS  =  -I../vslib $(RPM_OPT_FLAGS) $(CCDEF)
+CCFLAGS =  -I../vslib $(RPM_OPT_FLAGS) $(CCDEF)
 LDFLAGS = -L../vslib -lvslib $(LDDEF)
 ARFLAGS = 
 TARGET  = ftparc
diff -urN vfu-1.51.orig/ftparc/mm.conf vfu-1.51/ftparc/mm.conf
--- vfu-1.51.orig/ftparc/mm.conf	Sat Aug 14 17:03:15 1999
+++ vfu-1.51/ftparc/mm.conf	Sat Aug 14 17:05:30 1999
@@ -1,8 +1,8 @@
 
 $CC      = "g++";
 $LD      = "g++";
-$CCFLAGS = "-I../vslib -O2     \$(CCDEF)";
+$CCFLAGS = "-I../vslib $(RPM_OPT_FLAGS) \$(CCDEF)";
 $LDFLAGS = "-L../vslib -lvslib \$(LDDEF)";
 $TARGET  = "ftparc";
 $SRC     = "*.cpp";
-$HDR     = "*.h";
\ No newline at end of file
+$HDR     = "*.h";
diff -urN vfu-1.51.orig/vfu/makefile vfu-1.51/vfu/makefile
--- vfu-1.51.orig/vfu/makefile	Sat Aug 14 17:03:15 1999
+++ vfu-1.51/vfu/makefile	Sat Aug 14 17:05:56 1999
@@ -3,8 +3,8 @@
 CC      = g++
 LD      = g++
 AR      = ar rvs
-CFLAGS  =  -I../vslib -I/usr/include/ncurses -O2 $(CCDEF)
-CCFLAGS =  -I../vslib -I/usr/include/ncurses -O2 $(CCDEF)
+CFLAGS  =  -I../vslib -I/usr/include/ncurses $(RPM_OPT_FLAGS) $(CCDEF)
+CCFLAGS =  -I../vslib -I/usr/include/ncurses $(RPM_OPT_FLAGS) $(CCDEF)
 LDFLAGS = -L../vslib -lvslib -lncurses $(LDDEF)
 ARFLAGS = 
 TARGET  = vfu
diff -urN vfu-1.51.orig/vslib/makefile vfu-1.51/vslib/makefile
--- vfu-1.51.orig/vslib/makefile	Sat Aug 14 17:03:15 1999
+++ vfu-1.51/vslib/makefile	Sat Aug 14 17:06:12 1999
@@ -3,8 +3,8 @@
 CC      = g++
 LD      = g++
 AR      = ar rvs
-CFLAGS  =  -I. -I/usr/include/ncurses -O2 $(CCDEF)
-CCFLAGS =  -I. -I/usr/include/ncurses -O2 $(CCDEF)
+CFLAGS  =  -I. -I/usr/include/ncurses $(RPM_OPT_FLAGS) $(CCDEF)
+CCFLAGS =  -I. -I/usr/include/ncurses $(RPM_OPT_FLAGS) $(CCDEF)
 LDFLAGS = $(LDDEF)
 ARFLAGS = 
 TARGET  = libvslib.a
diff -urN vfu-1.51.orig/vslib/mm.conf vfu-1.51/vslib/mm.conf
--- vfu-1.51.orig/vslib/mm.conf	Sat Aug 14 17:03:15 1999
+++ vfu-1.51/vslib/mm.conf	Sat Aug 14 17:06:20 1999
@@ -1,7 +1,7 @@
 
 $CC      = "g++";
 $LD      = "g++";
-$CCFLAGS = "-I. -I/usr/include/ncurses -O2 \$(CCDEF)";
+$CCFLAGS = "-I. -I/usr/include/ncurses $(RPM_OPT_FLAGS) \$(CCDEF)";
 $LDFLAGS = "\$(LDDEF)";
 $TARGET  = "libvslib.a";
 $SRC     = "*.cpp";
